contrib/podman-next/fcos-podmanimage/Containerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/fedora/fedora-coreos:stable
+
+# Setup the podman-next copr repo
```

Comment:
hello, we wanted to have a specific tag pushed on the registry (in addition to a rolling tag like `latest`, `next`, whatever but I don't see expiration using `quay.expires-after` LABEL being set ?

https://access.redhat.com/documentation/fr-fr/red_hat_quay/3/html/use_red_hat_quay/working_with_tags#setting_tag_expiration_from_a_dockerfile

images won't expire ?

Reply:
What would the tag be? The short commit ID maybe?

Reply:
ack, i'll add those too, thanks

Reply:
Interesting, I didn't know about the `quay.expires-after` mechanism, assumed you had to set that via their API.  Neat!  Should that be something like +6months or something else?

Reply:
short I'd would be fine, about expiry time I have no strong opinion. but I'm not sure I will try to get development versions before a major release.
So if you release every 3 months then it might be the good window

Reply:
i'm setting expiry to 30d for now. Regarding tags, looks like those can be handled in the GHA file.